tests/js/sentry-test/modal.jsx

```diff
@@ -2,10 +2,23 @@ import {mountWithTheme} from 'sentry-test/enzyme';
 
 import GlobalModal from 'sentry/components/globalModal';
 
+const mountedModals = [];
+
 export async function mountGlobalModal(context) {
   const modal = mountWithTheme(<GlobalModal />, context);
+  mountedModals.push(modal);
   await tick();
   modal.update();
 
   return modal;
 }
+
+afterEach(() => {
+  while (mountedModals.length) {
+    const modal = mountedModals.pop();
+    if (modal.length === 1) {
```

Comment:
What's the type of `modal` here, does it have a length? Should we just check if it exists: `if (modal)`?

Reply:
I swapped it to `modal.exists()`:
> Returns whether or not any nodes exist in the wrapper. Or, if a selector is passed in, whether that selector has any matches in the wrapper.

I guess the length check returned the number of nodes, which is mostly 1, but this feels a bit nicer.

Reply:
Returns whether or not any nodes exist in the wrapper. Or, if a selector is passed in, whether that selector has any matches in the wrapper.